ipyvolume/widgets.py

```diff
@@ -64,6 +64,28 @@ def _default_material(self):
     def _default_line_material(self):
         return pythreejs.ShaderMaterial()
 
+    @traitlets.observe('color')
+    def _observe_color(self, change):
+        '''Checks dimentions of the array of colors and adds alpha channel if needed.'''
+        new_color = change['new']
+
+        if type(new_color) == np.ndarray:
+            # two dimensitional array
+            if len(new_color.shape) == 2:
+                rows, cols = new_color.shape
+                # add alpha channel if needed
+                if cols == 3:
+                    color_alpha = np.ones((rows, 4))
+                    color_alpha[:, :-1] = new_color
+                    self.color = color_alpha
+
+            elif (len(new_color.shape) == 1) and (new_color.shape[0] == 3):
+                # we have one row, without alpha channel, we'll add it here
+                color_alpha = np.ones(4)
+                color_alpha[:-1] = new_color
+                self.color = color_alpha
+
```

Comment:
I'm not sure about this one, I have a preference for not changing the trait on the Python side, but only at the moment it goes into the Attribute at the js side, or as an intermediate solution, do this transformation in serialize.py::color_to_binary_or_json.
The reason for this is that eventually, we want to have things on the js side linked to other libraries, that may not support alpha.